### PR TITLE
gce 'getting started' cleanup.

### PR DESCRIPTION
**Pull request by amygdala**

### CODE REVIEW

**Comment by amygdala** [comment]
Addresses #9632 for GCE. Also partially addresses #9420, by giving some sanity-check kubectl commands on cluster startup.  Adds a TOC (LMK what you think of that) and expands on some of the sections.
If this looks good, can make similar mods to some of the other 'getting started' docs.
TJ, since #9632 was your bug, assigning to you, but feel free to unassign.

File: docs/getting-started-guides/gce.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
+
 ## Getting started on Google Compute Engine
 
+**Table of Contents**
+
+  - [Before you start](#before-you-start)
+  - [Prerequisites](#prerequisites)
+  - [Starting a Cluster](#starting-a-cluster)
```
Comment (goltermann) [inline docs/getting-started-guides/gce.md]: Lowercase "c" in cluster to conform with the rest of the list

Comment (amygdala) [inline docs/getting-started-guides/gce.md]: Done, thanks.

**Comment by goltermann** [comment]
LGTM.  You might want to squash your commits before we merge.

**Comment by amygdala** [comment]
squashed.

**Pull request merged**